cEP-0010.md

```diff
@@ -0,0 +1,290 @@
+clones or more specifically very similar blocks of code in your codebase.
+* `Root.Redundancy.Clone.min_clone_tokens`(a taste of the `Clone` aspect):
+Thsi taste represents the minimun number of token that have to be equal to be
+considered as a clone.
+* `Root.Redundancy.UnusedImport`: This taste describes useless import/include
+that is not needed.
+* `Root.Redundancy.UnreachableCode`: This taste describes source code that
+can never be executed during the program execution.
+* `Root.Redundancy.UnusedFunction`: This taste describes functions that
+are never called during code execution
+* `Root.Redundancy.UnreachableStatement`: This taste describes statement that
+are never executed during code execution.
+* `Root.Redundancy.UnusedParameter`: This taste describes functions'
+parameters which are never used.
+* `Root.Redundancy.UnusedLocalVariable`: This taste describes variables which
+are defined locally but never used.
+* `Root.Redundancy.UnusedGlobalVariable`: this taste describes variables which
+have a global scope but are never used.
+* `Root.Metadata`: This describes any aspect that is related to metadata that
+is not inside your source code.
+* `Root.Metadata.CommitMessage`.
+* `Root.Metadata.CommitMessage.Emptiness`: This aspect is to detect and
+prevent from using empty commit message.
+* `Root.Metadata.CommitMessage.Shortlog`: This aspect checks on the first line
+of your commit message.
+* `Root.Metadata.CommitMessage.Shortlog.ColonExistence`: This aspect checks
+your commit shortlog to see whether or not there is a colon (given that the
+usage of colon is compulsory in your project).
+* `Root.Metadata.CommitMessage.Shortlog.TrailingPeriod`: This aspect checks
+for trailing period.
+* `Root.Metadata.CommitMessage.Shortlog.Tense`: This aspect checks on the
+tense used in commit message shortlog.
+* `Root.Metadata.CommitMessage.Shortlog.Length`: This aspect checks on the
+length of your commit message shortlog given the value of
+`Root.Metadata.CommitMessage.Shortlog.Length.max_shortlog_length`(Taste obj).
+* `Root.Metadata.CommitMessage.Shortlog.FirstCharacter`: This aspect detects
+inconsistent casing for the first character of your commit message shortlog(
+and the the first character following the colon if there is one) given the
+value of
+`Root.Metadata.CommitMessage.Shortlog.FirstCharacter.shortlog_starts_upper_case`
+[(Taste obj) representing the case to use for the first character of your
+commit message shortlog and the first character following the colon].
+* `Root.Metadata.CommitMessage.Body`: Parent aspect for all of the aspects
+related to your commit message body.
+* `Root.Metadata.CommitMessage.Body.Existence`: Checks on the existence of
+your commit message body (it enforces its usage).
+* `Root.Metadata.CommitMessage.Body.Length`: Checks on the length of your
+commit message body given the value of
+`Root.Metadata.CommitMessage.Body.Length.max_body_length`(Taste obj).
```

Comment:
IMO we should just link to aspect docs here later instead of describing every aspect and taste here again